src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/CommandID.cs

```diff
@@ -94,7 +82,7 @@ public virtual Guid Guid
         /// </summary>
         public override string ToString()
         {
-            return _menuGroup.ToString() + " : " + _commandID.ToString(CultureInfo.CurrentCulture);
+            return $"{_menuGroup.ToString()} : {_commandID.ToString(CultureInfo.CurrentCulture)}";
```

Comment:
Same comment.  In general, we should be careful about using string interpolation in anything other than debug code.  In some cases it may make sense, but string.Concat with string inputs is faster, especially when the number of input strings is <= 4, as there are string.Concat overloads for handling those cases explicitly (and the compiler maps string concatenation to using string.Concat).